### PR TITLE
Updated action styles to v2, and overhauled Stake action

### PR DESCRIPTION
**Pull request by NoahSaso**
This updates the actions to fit the v2 designs. For example:

Screenshot 2022-11-11 at 1 18 32 PM

This also completely overhauls the Stake action to allow selecting from a validator instead of requiring manual address entry. It also correctly validates the balances and displays those.

## Stake
Screenshot 2022-11-11 at 2 18 10 PM

## Unstake
Screenshot 2022-11-11 at 2 31 55 PM

## Restake
Screenshot 2022-11-11 at 2 19 13 PM
Screenshot 2022-11-11 at 2 19 32 PM

## Claim
Screenshot 2022-11-11 at 2 19 45 PM

### Before execution
Screenshot 2022-11-11 at 2 25 34 PM

### After execution
Screenshot 2022-11-11 at 2 22 22 PM

## Choose validator
Screenshot 2022-11-11 at 1 21 55 PM
Screenshot 2022-11-11 at 2 20 47 PM
Screenshot 2022-11-11 at 1 22 49 PM

### CODE REVIEW

**Comment by NoahSaso** [comment]
Moved the treasury balance / delegated amount down because it looked weird.

Screenshot 2022-11-11 at 1 27 57 PM
Screenshot 2022-11-11 at 1 28 24 PM

File: packages/i18n/locales/en/translation.json [hunk 1]
```diff
@@ -153,15 +155,22 @@
...
     "noClaimsAvailable": "No claims available.",
+    "noCw20Tokens": "There are no CW20 tokens in the treasury.",
     "noMembers": "You haven't added any members to your DAO.",
     "noNftsSelected": "No NFTs selected.",
+    "noValidatorFound": "No validator found.",
     "noVotingPower": "You have not given anyone voting power. Add some members to your DAO.",
     "notCw20Address": "This contract address is not a CW20 token.",
     "notCw721Address": "This contract address is not a NFT (CW721).",
     "notEnoughForDeposit": "You do not have enough tokens in your balance to pay the proposal deposit",
+    "notInActiveSet": "Not in active set.",
```
Comment (elsehow) [inline packages/i18n/locales/en/translation.json:166]: Consider: "This validator is not in the active set."

File: packages/state/recoil/selectors/chain.ts [hunk 2]
```diff
@@ -478,7 +486,7 @@ export const nativeStakingInfoSelector = selectorFamily<
 })
 
 export const transactionEventsSelector = selectorFamily<
-  Record<string, any>[],
+  readonly Event[] | undefined,
```
Comment (elsehow) [inline packages/state/recoil/selectors/chain.ts:489]: Had to Google this `readonly` property - this is great!

File: packages/stateful/actions/components/Stake.stories.tsx [hunk 3]
```diff
@@ -12,26 +13,106 @@ export default {
...
+    },
+    rewards: 1.23,
+    ...denomProps,
+  },
+  {
+    // Random price between 0 and 10000 with up to 6 decimals.
+    amount: Math.floor(Math.random() * (10000 * 1e6) + 1e6) / 1e6,
+    validator: {
+      address: 'elsehow',
+      moniker: 'elsehow',
```
Comment (elsehow) [inline packages/stateful/actions/components/Stake.stories.tsx:63]: Flattered by this story.

File: packages/stateful/actions/components/ValidatorPicker.tsx [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+  // Map validator address to staked amount.
+  const validatorStakedAmountMap = stakes?.reduce(
+    (acc, stake) => ({
+      ...acc,
+      [stake.validator.address]: stake.amount,
+    }),
+    {} as Record<string, number | undefined>
+  )
+
+  // Sort staked first, then by total staked tokens (i.e. voting powe and
```
Comment (elsehow) [inline packages/stateful/actions/components/ValidatorPicker.tsx:60]: Typo: power

File: packages/stateful/actions/components/ValidatorPicker.tsx [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+  // popularity).
+  const sortedValidators = [...validators].sort((a, b) => {
+    const aStake = validatorStakedAmountMap?.[a.address]
```
Comment (elsehow) [inline packages/stateful/actions/components/ValidatorPicker.tsx:63]: What's going on here? This is a bit of a mysterious one-liner to me. I'm sure it gets the job done, but a comment might help a future contributor.

File: packages/stateful/actions/components/ValidatorPicker.tsx [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+          moniker,
+          details,
+          website,
+          commission,
+          tokens,
+          status,
+        } = validator
+
+        const existingStake = stakes?.find(
+          (stake) => stake.validator.address === address
```
Comment (elsehow) [inline packages/stateful/actions/components/ValidatorPicker.tsx:126]: Love this

**Review by elsehow — APPROVED** [review]
Looks great (: Some small comments here and there.

**Pull request merged**